### ISSUE TITLE
Double quotes in column descriptions not escaped

### ISSUE THREAD

**Issue by danny-chain** [datnguye/dbterd#36]
**Describe the bug**
Double quotes in a column description in schema.yml produces outbut.dbml that does not parse because the ""s are not \ escaped. 

**To Reproduce**
1. Create any column description with quotes, e.g:
```
- name: my_col
  description: >-
    Here is "my" test column
```
2. Observe output in output.dbml. Output is:
`"my_col" "character varying(256)" [note: "Here is "my" test column"]`

**Expected behavior**
Output in output.dbml should be:
`"my_col" "character varying(256)" [note: "Here is \"my\" test column"]`

**Desktop (please complete the following information):**
 - OS: mac
 - Version: dbterd v1.2.0

**Additional context**
I'm not positive that the backslash-escaped quotes is a requirement of the .dbml spec, however adding them allowed my consuming app to successfully load the output.dbml without throwing a parse error.

**Comment by datnguye** [comment]
Thanks @danny-chain 
It is a bit supprising by this issue, but it was my bad where not to implement the test on this case. Will work on it this weekend. Cheers

**Comment by datnguye** [comment]
Fixed in [1.2.3](https://pypi.org/project/dbterd/1.2.3/)
